src/Symfony/Bridge/Doctrine/Messenger/DoctrinePingConnectionMiddleware.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Doctrine\Messenger;
+
+use Doctrine\Common\Persistence\ManagerRegistry;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Messenger\Envelope;
+use Symfony\Component\Messenger\Middleware\MiddlewareInterface;
+use Symfony\Component\Messenger\Middleware\StackInterface;
+
+/**
+ * Wraps all handlers in a single doctrine transaction.
+ *
+ * @author Tobias Nyholm <insidestyles@gmail.com>
```

Comment:
Is it true?

//cc @Nyholm

Reply:
No, but this is similar to an other middleware I wrote. I guess it is just a copy/paste error. 

=)